pages/index.vue

```diff
@@ -30,12 +30,14 @@
 export default {
   data () {
     return {
-      usecases: {},
+      usecaseItems: {},
+      usecase0: {},
       error: false
     }
   },
   async mounted () {
-    this.usecases = await this.$content('usecaseItems').fetch()
+    this.usecaseItems = await this.$content('usecaseItems').fetch()
+    this.usecase0 = await this.$content('usecase_0').sortBy('slug', 'asc').fetch()
```

Comment:
This is great already! I suppose we want to get rid of the hardcoded `usecase0` eventually, right? Then we can loop over all usecases in `usecaseItems`. Do you think we can do that already, or shall we postpone it till later?

Reply:
Thanks for your comment Peter! This is indeed something I would like to discuss with you. I propose that we could postpone it till later. We will definitely remove the hardcoded `usecase0` and I think it won't be difficult to write a loop for it. The thing is, with multiple usecases, we may need to adjust the styling, together with the `tag` panel which we might need to add later. When we have multiple usecases in place, it would be easier for testing. What do you think?